Common/src/fem_wall_distance.cpp

```diff
@@ -277,31 +310,30 @@ void CMeshFEM_DG::ComputeWall_Distance(CConfig *config) {
 
         /* Check for an empty tree or a viscous wall.
            In those case the wall distance is set to zero. */
-        if(WallADT.IsEmpty() || viscousWall) {
+        if(viscousWall) {
 
           /* Wall distance must be set to zero. */
           for(unsigned short i=0; i<nInt; ++i)
             surfElem[l].wallDistance[i] = 0.0;
         }
-        else {
-
-          /*--- Not a viscous wall boundary, while viscous walls are present. 
+        else if ( !WallADT->IsEmpty() ) {
+          /*--- Not a viscous wall boundary, while viscous walls are present.
                 The distance must be computed. Loop over the integration points
                 and do so. ---*/
           for(unsigned short i=0; i<nInt; ++i) {
 
             const su2double *coor = surfElem[l].coorIntegrationPoints.data()
-                                  + i*nDim;
+                + i*nDim;
```

Comment:
This looks a bit awkward, all in one line or maybe even `&surfElem[l].coorIntegrationPoints[i*nDim]`

Reply:
Here again, just used the old implementation. Maybe would have been easier, if I just would have done an in-place modification of this routine. But I can change that anyway.

Reply:
Yup I know this is just moved around, but it caught my eye.